docs_src/microservices/application/services/AppRecordReplay/GettingStarted.md

```diff
@@ -0,0 +1,214 @@
+2. Core EdgeX services
+3. Application or Supporting service that will process the Events
+    - This guide uses the standard App Rules Engine as the app service which is processing the events.
+
+### Running Services
+
+The simplest way to run all the required services is to use the [Compose Builder](https://github.com/edgexfoundry/edgex-compose/tree/{{edgexversion}}/compose-builder) tool from a terminal window 
+
+1. Clone [https://github.com/edgexfoundry/edgex-compose/tree/{{edgexversion}}](https://github.com/edgexfoundry/edgex-compose/tree/{{edgexversion}})
+
+2. Change to the **compose-builder** folder
+
+3. Run the services
+    ```
+    make run no-secty ds-virtual as-record-replay
+    ```
+
+This runs, in non-secure mode, all the standard EdgeX services along with the Device Virtual and App Record Replay services.  App Rules Engine is part of the standard services and is used in this guide for processing the events.
+
+### Postman
+
+A sample PostMan collection is provided to simplify controlling this service via its REST API. See the [Postman Collection](../ApiReference/#postman-collection) section for more details.
+
+### Debug Logging
+
+Optionally, the debug logging can be enabled on the **App Record Replay** service. This is accomplish by setting `LogLevel` value via the Consul UI at [http://localhost:8500/ui/dc1/kv/edgex/v3/app-record-replay/Writable/LogLevel/edit](http://localhost:8500/ui/dc1/kv/edgex/v3/app-record-replay/Writable/LogLevel/edit) to `DEBUG`
+
+App service debug logging is very verbose, so when viewing the logs for this service. It is useful to filter the log messages for those that start with "ARR ". If you are using **Portainer** to view the service logs, simply place the "ARR " text in the `search` text box.
+
+To see that the replayed events are being "processed" we also need to enable debug logging for **App Rules Engine**. Set the `LogLevel` value via the Consul UI at [http://localhost:8500/ui/dc1/kv/edgex/v3/app-rules-engine/Writable/LogLevel/edit](http://localhost:8500/ui/dc1/kv/edgex/v3/app-rules-engine/Writable/LogLevel/edit) to `DEBUG`
+
+For **App Rules Engine** we want filter for messages showing the events received. Filtering the log messages for "match the incoming topic" text will accomplish this.
+
+### Recording a Session
+
+#### Start a Recording
+
+Before starting a recording session first review the **Start Recording** POST API in the [API Reference](ApiReference.md) section. Be sure to view both examples.
+
+The source device service(s) need to be producing events prior to or shortly after the recording is starting. In this guide, Device Virtual is already producing events. So all that is need is to start the recording session by using the `Start Recording` request from the PostMan collection referenced above. Edit the `RecordRquest` to set the `duration` and/or `eventLimit` and optionally any of the filters. The existing example filters can be removed so that all events are recorded.
```

Comment:
```suggestion
The source device service(s) need to be producing events prior to or shortly after the recording is starting. In this guide, Device Virtual is already producing events. So all that is need is to start the recording session by using the `Start Recording` request from the PostMan collection referenced above. Edit the `RecordRequest` to set the `duration` and/or `eventLimit` and optionally any of the filters. The existing example filters can be removed so that all events are recorded.
```

Reply:
fixed